Scripts/install_swiftlint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+# Run pod install to fetch the SwiftLint binary
+pod install
+
+# Undo all the changes made by CocoaPods
+git checkout ReSwift.xcodeproj/project.pbxproj
```

Comment:
This could backfire, discarding other changes to this file before running the script and not committed.

I did play around with stashing before running `pod install`, but that can backfire as well, because if you don't have any changes and an empty stash, when you'll try to `pop` you'll get a "No stash found" error.

🤔